influence_benchmark/RL/base_iteration.py

```diff
@@ -134,29 +147,32 @@ def _save_kwargs(self, kwargs):
             yaml.dump(self.kwargs_to_save, outfile, default_flow_style=False)
 
     def create_environment_and_agent(
-        self, device, progress, shared_queue, agent_config, lora_path=None
+        self, agent_device, env_device, progress, shared_queue, agent_config, lora_path=None
     ) -> Tuple[VectorizedEnvironment, Agent]:
         agent_backend_class = model_name_to_backend_class(self.agent_model_name)
         env_backend_class = model_name_to_backend_class(self.env_model_name)
         env_backend = env_backend_class(
             model_name=self.env_model_name,
             model_id=self.agent_model_id,  # type: ignore
-            device=device,
-            lora_path=lora_path,
+            device=env_device,
+            lora_path=None,  # The environment is never finetuned
```

Comment:
wow were we accidentally fine-tuning our environment models too?

If `self.agent_model_name == self.env_model_name`, doesn't that mean that we'd be setting the lora to None for the agent backend too?

Reply:
No, the lora is still disabled for all roles but the agent. Yes it seems like this would break the agent lora haha